rpi-eeprom-update

```diff
@@ -436,19 +442,20 @@ directory first. The binaries are then promoted to 'latest' and finally 'default
 so the 'default' binary is always the most tested release.
 
 default:
-The default bootloader image which is updated once new features in
-'latest' are stable or for critical hardware or security updates.
-
-Raspberry Pi OS automatically updates the bootloader when a newer 'default' image
-is available following an APT update to the rpi-eeprom package.
+The default bootloader image which supports all current models and hardware
+revisions.
+If a critical bug fix is required then the minimum default version number
+BOOTLOADER_AUTO_UPDATE_MIN_VERSION in the rpi-eeprom package is updated
+causing the bootloader to be automatically updated. Otherwise, if the 'default'
+bootloader release is selected then the bootloader is only updated if the
```

Comment:
"the bootloader is only updated if..." - is this text still relevant now that you've added the `-s` option?